uiowa/uiowa
# Feature: stat regression fix

This pull request resolves https://github.com/uiowa/uiowa/issues/2853.   This pr also includes a fix in `uids_base.libraries.yml` to adjust the weight of the `background.css` file to get this recent UIDS issue working https://github.com/uiowa/uids/issues/456 - you can view this update at http://sandbox.local.drupal.uiowa.edu/cards.  

# How to test

```
rm -rf ./node_modules
yarn cache clean
yarn install
yarn workspace uids_base gulp --development
blt ds --site=admissions.uiowa.edu
```
1. Check that you have `3.1.8` of UIDS in your node_modules folder, I was having trouble locally getting this to `yarn install`. 
3. Test that stats on the home page are centered and that the hover/no-hover layout builder style is working. 
4. Test that jumpiness on stat grid on https://admissions.prod.drupal.uiowa.edu/ is fixed.
